leaderboards/static/js/src/grade_leaderboard_studio.js

```diff
@@ -0,0 +1,33 @@
+/* Javascript for GradeLeaderboardXBlock. */
+function GradeLeaderboardStudioXBlock(runtime, element) {
+    "use strict";
+    $('.save-button', element).bind('click', function(e) {
+        e.preventDefault();
+        var data = {
+            graded_target_id: $(element).find('#graded_target_id').val(),
+            count: $(element).find('input[name=count]').val()
+        };
+        runtime.notify('save', {state: 'start', message: gettext("Saving")});
+        $.ajax({
+            type: "POST",
+            url: runtime.handlerUrl(element, 'studio_submit'),
+            data: JSON.stringify(data),
+            dataType: "json",
+            global: false,  // Disable Studio's error handling that conflicts with studio's notify('save') and notify('cancel') :-/
+            success: function(response) { runtime.notify('save', {state: 'end'}); }
+        }).fail(function(jqXHR) {
+            var message = gettext("This may be happening because of an error with our server or your internet connection. Try refreshing the page or making sure you are online.");
```

Comment:
I still hate that we're reinventing the wheel here-- I know you copied that error message from upstream, after all. If there's not a better way to do this, we really ought to create a contribution upstream.

XBlocks often rely heavily on AJAX. Doing semantic AJAX should not be this painful.